database/db/trimmed_tables/_DensityUnits/init_density_units.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS temp_density_units (
+  Id INT NOT NULL,
+  Name VARCHAR(100) NOT NULL,
+  ConvFactor FLOAT NOT NULL,
+  PRIMARY KEY (Id)
+);
+\copy temp_density_units (Id, Name, ConvFactor) from 'docker-entrypoint-initdb.d/_DensityUnits.csv' with header delimiter ',' CSV ;
+SELECT * INTO density_units
+FROM temp_density_units
+ALTER TABLE density_units
```

Comment:
These trimmed table scripts don't need the ALTER TABLE and DROP COLUMN lines, or to copy into a temp table. You can look in the other trimmed table scripts for examples.